elyra/pipeline/airflow/provider_package_catalog_connector/airflow-provider-package-catalog.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
+  "$id": "https://raw.githubusercontent.com/elyra-ai/elyra/master/catalog-connectors/airflow/airflow-provider-package-catalog-connector/airflow-provider-package-catalog.json",
+  "title": "Apache Airflow provider package operator catalog",
+  "name": "airflow-provider-package-catalog",
+  "schemaspace": "component-catalogs",
+  "schemaspace_id": "8dc89ca3-4b90-41fd-adb9-9510ad346620",
+  "metadata_class_name": "elyra.pipeline.component_metadata.ComponentCatalogMetadata",
+  "uihints": {
+    "title": "Apache Airflow provider package operator catalog",
+    "icon": "",
+    "reference_url": "https://github.com/elyra-ai/elyra/tree/master/catalog-connectors/airflow/airflow-provider-package-catalog-connector"
```

Comment:
And similar comment as above re: path to the reference url once documentation is updated as well